avm/res/resources/deployment-script/tests/e2e/defaults/main.test.bicep

```diff
@@ -0,0 +1,68 @@
+targetScope = 'subscription'
```

Comment:
Instead of removing both the CLI + PS tests from vanilla CARML, I'd suggest to also keep them in addition to the ones you added here. This has the benefit that those are added as tangible examples :)

Reply:
Should I follow the same folder structure in CARML ; a folder for CLI and one for PS?

Reply:
That would be nice.. Then we'd have 5 folders total, including 2 practical examples